docs/orleans/streaming/streams-quick-start.md

```diff
@@ -115,7 +117,7 @@ TimeSpan.FromMilliseconds(1_000));
 
 ## Subscribing and receiving streaming data
 
-For receiving data, we can use implicit/explicit subscriptions, which are fully described in other pages of the manual. Here we use implicit subscriptions, which are easier. When a grain type wants to implicitly subscribe to a stream, it uses the attribute [[ImplicitStreamSubscription (namespace)]](xref:Orleans.ImplicitStreamSubscriptionAttribute).
+For receiving data, we can use implicit/explicit subscriptions, which are fully described in other pages of the manual. Here we use implicit subscriptions, which are easier. When a grain type wants to implicitly subscribe to a stream, it uses the attribute [[ImplicitStreamSubscription(namespace)]](xref:Orleans.ImplicitStreamSubscriptionAttribute).
```

Comment:
```suggestion
For receiving data, you can use implicit/explicit subscriptions, which are fully described in other pages of the docs. This example uses implicit subscriptions, which are easier. When a grain type wants to implicitly subscribe to a stream, it uses the attribute [[ImplicitStreamSubscription(namespace)]](xref:Orleans.ImplicitStreamSubscriptionAttribute).
```

Reply:
How about "For receiving data, use..." - just curious if we are keen to avoid direct references to the reader? (yes, this could be chalked up as personal preference)

Reply:
Either way is probably fine (just not "we"). The style guide says to have a "conversational tone". Also see https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=25524.